dds/idl/be_global.h

```diff
@@ -260,6 +266,7 @@ class BE_GlobalData {
   bool root_default_nested_;
   bool warn_about_dcps_data_type_;
   ExtensibilityKind default_extensibility_;
+  bool default_enum_extensibility_zero_;
```

Comment:
When this is `false`, is there another config option to set the default enum extensibility to final or appendable?

Reply:
No.  Appendable is the default.   I think we are asking the user to use the annotations.